eng/versioning/version_client.txt

```diff
@@ -135,9 +135,9 @@ com.microsoft.azure:spring-cloud-starter-azure-appconfiguration-config;1.2.8-bet
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
-unreleased_com.azure:azure-core;1.9.0-beta.1
+unreleased_com.azure:azure-core;1.9.0
 unreleased_com.azure:azure-core-amqp;1.6.0-beta.1
-unreleased_com.azure:azure-core-test;1.5.0-beta.1
+unreleased_com.azure:azure-core-test;1.5.0
```

Comment:
Should we add PR description for the `unreleased_com` dependencies and listing out packages that are being updated for a quick look?